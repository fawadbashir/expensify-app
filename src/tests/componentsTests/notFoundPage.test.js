import React from 'react'
import { shallow } from 'enzyme'
import {createSerializer} from 'enzyme-to-json'
import NotFoundPage from '../../Components/NotFoundPage'

expect.addSnapshotSerializer(createSerializer({mode: 'shallow'}));

test('should Render Not Found Expense Page', () => {
    const wrapper = shallow(<NotFoundPage/>)
    expect(wrapper).toMatchSnapshot()
})