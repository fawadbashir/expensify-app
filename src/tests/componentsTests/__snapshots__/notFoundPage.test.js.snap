// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should Render Not Found Expense Page 1`] = `
<div>
  404 - 
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
